gateways/istio-egress/templates/deployment.yaml

```diff
@@ -197,12 +197,6 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          {{- if $gateway.sds }}
-          {{- if $gateway.sds.enabled }}
```

Comment:
`gateway.sds.enabled` used for public facing certificate for ingress. From the doc I can tell, https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/

We do support SDS, for egress gateway, for workload identity, between workload and egress, but that's governed already by `if .Values.global.sds.enabled`  below.

I can't think of a use case to support https cert on egress, seems abnormal. 

@JimmyCYJ WDTY

Reply:
@incfly We don't support SDS for external traffic at egress. There is a tracking GH issue https://github.com/istio/istio/issues/14039.

Reply:
Thanks confirmation. Approving.